commission/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Commissions",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "depends": ["product", "sale_management"],
```

Comment:
```suggestion
    "depends": ["product", "account"],
```

It shouldn't depend on sale_management.  Menu items should be moved under invoicing.